packages/terra-list/src/ListSection.jsx

```diff
@@ -22,9 +22,9 @@ const propTypes = {
    */
   isCollapsible: PropTypes.bool,
   /**
-   * Optionally sets the heading level. One of `1`, `2`, `3`, `4`, `5`, `6`.
+   * Optionally sets the heading level. One of `2`, `3`, `4`, `5`, `6`.
    */
-  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
+  level: PropTypes.oneOf([2, 3, 4, 5, 6]),
```

Comment:
Update defaultProps.level to 2

Reply:
Done [c8c28cc](https://github.com/cerner/terra-core/pull/3748/commits/c8c28cc0b0a77f76f4fb59b45af2eb150b139ac6)